csi/util.go

```diff
@@ -252,9 +270,9 @@ func ensureMountPoint(targetPath string, mounter mount.Interface) (bool, error)
 
 	IsCorruptedMnt := mount.IsCorruptedMnt(err)
 	if !IsCorruptedMnt {
-		logrus.Debugf("mount point %v try reading dir to make sure it's healthy", targetPath)
-		if _, err := os.ReadDir(targetPath); err != nil {
```

Comment:
Can't this detect a corrupted fs?

Reply:
When volume detached and re-attached and pod restarts quickly, it still can read the directory as normal.
OS also considers the mount point is valid at that moment.

Reply:
But yes, it could detect a corrupted fs after mount point became invalid.

Reply:
OK. Got it. You need to trigger IO write via `sync` later hence it's better to directly open the path then read the dir via the fd.